ansys/dpf/core/plotter.py

```diff
@@ -376,10 +418,15 @@ def add_mesh(self, meshed_region, **kwargs):
         >>> pl.add_mesh(mesh)
 
         """
-        self._internal_plotter.add_mesh(meshed_region=meshed_region, **kwargs)
+        self._internal_plotter.add_mesh(meshed_region=meshed_region,
+                                        scaling_result=scaling_result,
+                                        scaling_factor=scaling_factor,
+                                        **kwargs)
 
     def add_field(self, field, meshed_region=None, show_max=False, show_min=False,
-                  label_text_size=30, label_point_size=20, **kwargs):
+                  label_text_size=30, label_point_size=20,
+                  scaling_result=None, scaling_factor=1.0,
```

Comment:
I don't understand why we have scaling_result both for add_mesh, add_field and show_figure

Reply:
@cbellot000 It is in add_mesh and add_field because when we plot a bare mesh, only add_mesh is called and I still wanted to give the possibility to deform it.
As for show_figure, it is not in the arguments.
It is however in Plotter.plot_contour

Reply:
makes sense, sorry for the show_figure, it's because of how github show difference on large files